config/twitter_keys.EXAMPLE.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Obviously to get this to work you need to add your twitter api details here, and remove the `.EXAMPLE` from this file. If there's a better way of doing this without creating this stuff just let me know and I'll change it :-)